core/wiki/macros/sliders.tid

```diff
@@ -0,0 +1,89 @@
+	tv-set-centralised="no"
+	storyLeftTiddler={{{ [<storyLeftTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storyleft-]] }}}
+	storyRightTiddler={{{ [<storyRightTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storyright-]] }}}
+	sidebarWidthTiddler={{{ [<sidebarWidthTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/rightwidth-]] }}}
+	tiddlerWidthTiddler={{{ [<tiddlerWidthTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/tiddlerwidth-]] }}}
+	storyWidthTiddler={{{ [<storyWidthTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storywidth-]] }}}
+	storyMinWidthTiddler={{{ [<storyMinWidthTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storywidth-]] }}}
+	sidebarMinWidthTiddler={{{ [<sidebarMinWidthTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storyminwidth-]] }}}
+	storyPaddingLeftTiddler={{{ [<storyPaddingLeftTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storypaddingleft-]] }}}
+	storyPaddingRightTiddler={{{ [<storyPaddingRightTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storypaddingright-]] }}}>
+
+	<$transclude
+		$variable="editor-resizer"
+		template=<<template>>
+		sliderCondition=<<sliderCondition>>
+		eventCatcherClass="tc-editor-preview-eventcatcher"
+		mode=<<mode>>
+	/>
+
+</$let>
+\end
+
+\procedure two-cell-slider(width:"100%",minHeight:"10px",templateLeft:"",templateRight:"",mode:"block",sliderWidth:"12px",padding:"12px",sliderCondition:"yes",leftMinWidth:"100px",rightMinWidth:"100px")
+\import $:/core/procedures/sidebar-resizer
+\function get.resizer.style.position() absolute
+\function get.template-right.width() [<sidebarWidthTiddler>get[text]] :else[[50%]]
+\function get.resizer.style.left() [[calc(100% - ]addsuffix<get.template-right.width>addsuffix[)]]
+\function get.resizer.style.width() [<sliderWidth>]
+\function set.throttling() no
+\function get.left.minwidth() [<leftMinWidth>]
+\function get.right.minwidth() [<rightMinWidth>]
+\function drag.direction.reverse() no
+\whitespace trim
+
+<div
+	style.width=<<width>>
+	style.min-height=<<minHeight>>
+	style.display="flex"
+	style.flex-direction="column"
+	style.padding-left=<<padding>>
+	style.padding-right=<<padding>>
+	style.padding-top=<<padding>>
+	style.padding-bottom=<<padding>>
+>
+
+	<$let
+		tv-set-storywidth-storyright="no"
+		tv-set-sidebarwidth="yes"
+		tv-set-centralised="no"
+		storyLeftTiddler={{{ [<storyLeftTiddler>!is[blank]] :else[<qualify>addsuffix[-]addsuffix<currentTiddler>sha256[]addprefix[$:/state/resizer/storyleft-]] }}}
```

Comment:
In the `:else` path the first `addsuffix[-]` after `<qualify>` is redundant, since it is turned into a number with `sha256[]`

So removing the first `addsuffix[-]` should make the string a little bit shorter